### ISSUE TITLE
Retina U-Net Segmentation Loss Not Decreasing

### ISSUE THREAD

**Issue by mibaumgartner** [MIC-DKFZ/medicaldetectiontoolkit#42]
Hi, thank your for this great repository! 
I encountered some problems during training Retina U-Net, similar to a previous issue. When Instance/Batch-norm is activated, the segmentation loss does not decrease and the segmentation does not work properly. This problem does not occur when no normalisation is used. After some digging into the code, I found a normalisation layer after the final convolution of the segmentation layer of Retina U-Net.

https://github.com/pfjaeger/medicaldetectiontoolkit/blob/2b343ba974a6936d10bc7ba2d571db28f0760df5/models/retina_unet.py#L378

I think there should be no normalisation of the segmentation logits (because it implicitly limits the output prediction) or did I miss something?

**Comment by pfjaeger** [comment]
Hi thanks for your feedback. Good catch! Although there are no hard rules on where to apply norm, I would intuitively agree that logits should enter the softmax unscaled. Please set norm to None here until I fix this in the next commit.  

Do you get better results with the "fixed norm" as opposed to no norm? Thing is even with other models where the norm was applied 'correctly' I did not see an improvement. I did not test this extensively though, recent neurips papers suggest the actual benefit of norm is to enable larger learning rates so I guess one would have to run some experiments on that. What data set are you working on?

**Comment by mibaumgartner** [comment]
I did some testing with Mask R-CNN, where I compared Instance Norm to no normalisation. The results were slightly worse when I removed the normalisation, but due to time constraints I was not able to do an exhaustive hyerparameter search. I guess the impact is somewhat small, but faster convergence speed and potentially higher learning rates are a nice thing to have :)
I'm working on an in-house dataset which is not public.
